### PR TITLE
Play random playlist entry / StreamInfo improvements

### LINKED ISSUE #83: Random playlist
Would it be possible to add the ability to play random video off a playlist?

### PR DESCRIPTION
**Pull request by theychx**
Gave me an opportunity to get rid of the lame ```first_entry``` stuff in ```StreamInfo```.

Closes #83

### CODE REVIEW

File: catt/stream_info.py [hunk 1]
```diff
@@ -68,17 +61,11 @@ def is_video(self):
 
     @property
     def is_playlist(self):
-        if not self.is_local_file:
-            return True if "entries" in self._preinfo else False
-        else:
-            return False
+        return True if not self.is_local_file and "entries" in self._preinfo else False
```
Comment (skorokithakis) [inline catt/stream_info.py]:
```python
return not self.is_local_file and "entries" in self._preinfo
```

**Comment by skorokithakis** [comment]
One minor comment, but I can merge anyway.

File: catt/stream_info.py (hunk 1 above)
Comment (theychx) [reply catt/stream_info.py]: Cheers. Why didn't I think of that?

Comment (skorokithakis) [reply catt/stream_info.py]: Oh it gets me every time too.

**Pull request merged**